src/pmdas/linux/mk.rewrite

```diff
@@ -43,6 +43,7 @@ metric disk.dev.write_merge { type -> $KERNEL_ULONG }
 metric kernel.all.sysfork { type -> $KERNEL_ULONG }
 metric kernel.all.running { type -> $KERNEL_ULONG }
 metric kernel.all.blocked { type -> $KERNEL_ULONG }
+metric kernel.all.btime { type -> U64 }
```

Comment:
This line isn't necessary, its only for if we'd previously got the metadata incorrect (but this is a new metric, so we'll get it right first time)